src/django/api/models.py

```diff
@@ -1023,6 +1024,9 @@ def get_approved_claim(self):
         return self.facilityclaim_set.filter(
             status=FacilityClaim.APPROVED).count() > 0
 
+    def current_tile_cache_key():
+        return format(Facility.objects.latest('updated_at').updated_at, 'U')
```

Comment:
I added this apparatus in d5a74ee

I wanted to assume that we could potentially use this in the future, so instead of a single row table with a `tile_version` column, the table has a `name` charfield that's required unique and an optional `version` int field for any version number.

I updated the method used to get the cache key to get it from the version column of the row whose name is `tile_version`, falling back to "0" if that's not available.

The associated migration has both the table creation & a data migration op to create a `tile_version` row.